spec/services/comments/destroy_comment_service_spec.rb

```diff
@@ -0,0 +1,49 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe Comments::Destroy do
+  subject(:comment_destroy) { described_class.call(params[:article_id], params[:id]) }
+
+  let(:article) { create(:article) }
+
+  let(:params) do
+    {
+      id: comment.id,
+      article_id: article.id
+    }
+  end
+
+  let!(:comment) { create(:comment, article:) }
+
+  describe '#call' do
+    context 'when success' do
+      it 'has not any error' do
+        expect(comment_destroy.errors[:not_found]).to be_nil
```

Comment:
`comment_destroy.errors` should be empty at all. If you're checking for success, then you need to be sure that you don't have any errors.